CHANGELOG.md

```diff
@@ -78,6 +78,8 @@
   * `-log.rate-limit-enabled`
   * `-log.rate-limit-logs-per-second`
   * `-log.rate-limit-logs-per-second-burst`
+* [ENHANCEMENT] Querier: cancel query requests to ingesters in a zone upon first error received from the zone, to reduce wasted effort spent computing results that won't be used #5764
```

Comment:
I'm confused by this line. Is this left over from something? The PR referenced is for the above changelog line.